packages/client/utils/parseWebPath.ts

```diff
@@ -0,0 +1,8 @@
+import {upperFirst} from './upperFirst'
+
+export const parseWebPath = (webPath: string) => {
```

Comment:
@mattkrick you mentioned in your CR ([here](https://github.com/ParabolInc/parabol/pull/6160#discussion_r829255475)) that we should just parse, rather than parse, filter, and concatenate. 

However, if we just parse and return `[fullPath, issuesAndIid]`, those values will always need to edited afterwards. Doing this in multiple places feels more error-prone that doing it all here imo